azdev/utilities/command.py

```diff
@@ -56,6 +55,34 @@ def cmd(command, message=False, show_stderr=True, **kwargs):
         return CommandResultItem(err.output, exit_code=err.returncode, error=err)
 
 
+def shell_cmd(command, message=False, stderr=None, stdout=None, check=True, raise_ex=True, timeout=None,
```

Comment:
The same question asked before, can this function `shell_cmd` be replaced by `cmd`?

Reply:
We need to keep it since it's necessary in insall_cli in venv.py and later updates in codegen.